_search-plugins/search-pipelines/using-search-pipeline.md

```diff
@@ -27,8 +27,32 @@ GET /my_index/_search?search_pipeline=my_pipeline
 For a complete example of using a search pipeline with a `filter_query` processor, see [`filter_query` processor example]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/filter-query-processor#example).
 
 ## Using a temporary search pipeline for a request
+1. You can provide a search pipeline id in the search request:
 
-As an alternative to creating a search pipeline, you can define a temporary search pipeline to be used for only the current query:
+```json
+GET /my-index/_search
+{
+    "query": {
+        "match_all": {}
+    },
+    "from": 0,
+    "size": 10,
+    "search_pipeline": "my_pipeline"
+}
+```
+
+or for msearch
```

Comment:
I don't know that this is a true "pattern", (and if it is we should have Vale enforce it), but in other places in the docs I see it referred to as "multi-search".   For example I see "All multi-search URL parameters are optional.".   
```suggestion
or for multi-search:
```